ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
 ETC_20150317_000097	$Yellow Griba
 ETC_20150317_000098	$Large Red Griba
 ETC_20150317_000099	$Big Griba
-ETC_20150317_000100	Someone once described this monster as hard to block and hard to eat.
+ETC_20150317_000100	$Someone once described this monster as hard to block and hard to eat.
 ETC_20150317_000101	$Big Blue Griba
-ETC_20150317_000102	A large loathsome creature that is intentionally hunted.
+ETC_20150317_000102	$Large, strong, and tough monster which is a reluctant prey of the hunters
```

Comment:
"Though it is large, strong, and tough, one feels guilty for hunting them." I believe this is a more accurate translation.